frontend/src/views/coverage/maintenance/ChangeEffectiveDate.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div id="changeEffectiveDate" v-if="updateMode">
+    <form @submit.prevent="submitForm">
+      <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.groupNumber" id="groupNumber" label="Group Number" type="text" v-model.trim="groupNumber" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.phn" id="phn" label="PHN" type="text" v-model.trim="phn" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col3">
+          <AppDateInput :e-model="v$.existingEffectiveDate" id="existingEffectiveDate" label="Existing Coverage Effective Date" v-model="existingEffectiveDate" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col3">
+          <AppDateInput :e-model="v$.newEffectiveDate" id="newEffectiveDate" label="New Coverage Effective Date" v-model="newEffectiveDate" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppButton :submitting="submitting" mode="primary" type="submit">Submit</AppButton>
+        <AppButton @click="resetForm" mode="secondary" type="button">Clear</AppButton>
+      </AppRow>
+    </form>
+  </div>
+  <div id="confirmation" v-if="updateOk">
+    <p>PHN: {{ result?.phn }}</p>
+    <AppButton @click="resetForm" mode="primary" type="button">Change Another Effective Date</AppButton>
+  </div>
+</template>
+
+<script>
+import MaintenanceService from '../../../services/MaintenanceService'
+
+import useVuelidate from '@vuelidate/core'
+import { validatePHN, validateGroupNumber, VALIDATE_PHN_MESSAGE, VALIDATE_GROUP_NUMBER_MESSAGE } from '../../../util/validators'
+import { required, helpers } from '@vuelidate/validators'
+import { API_DATE_FORMAT } from '../../../util/constants'
+import dayjs from 'dayjs'
+import { useAlertStore } from '../../../stores/alert'
+
+export default {
+  name: 'ChangeEffectiveDate',
+
+  setup() {
+    return {
+      alertStore: useAlertStore(),
+      v$: useVuelidate(),
+    }
+  },
+  data() {
+    return {
+      submitting: false,
+      updateOk: false,
+      updateMode: true,
+      phn: '',
+      groupNumber: '',
+      existingEffectiveDate: null,
+      newEffectiveDate: null,
+      result: {
+        phn: '',
+        status: '',
+        message: '',
+      },
+    }
+  },
+
+  methods: {
+    async submitForm() {
+      this.submitting = true
+      this.updateOk = false
+      this.updateMode = true
```

Comment:
I don't think you need two separate flags. You can just use a single flag and v-if/v-else. See the latest update to ReinstateOverAgeDependent.